ara/controller/pagination.py

```diff
@@ -0,0 +1,14 @@
+from typing import Any, Optional
+
+from pydantic import BaseModel
+
+
+class PaginatedData(BaseModel):
+    count: int
+    next: Optional[str]
```

Comment:
`Optional[str]` 대신 `str | None`을 사용합시다.
- [discuss.python.org: Clarification for PEP 604](https://discuss.python.org/t/clarification-for-pep-604-is-foo-int-none-to-replace-all-use-of-foo-optional-int/26945/10)
- [Łukasz Langa: PEP 563 and PEP 649](https://lukasz.langa.pl/61df599c-d9d8-4938-868b-36b67fdb4448/)

```suggestion
    next: str | None
```

Reply:
역시 최신 python!!!! 👍